### PR TITLE
Community application

### PR DESCRIPTION
**Pull request by harjaapdhillon16**

### CODE REVIEW

File: src/components/pages/landing/communityApplicationPanel/index.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    register,
+    handleSubmit,
+    formState: { errors },
+    reset,
+  } = useForm();
+  const [formLoading, setFormLoading] = useState(false);
+  const [previewImage, setPreviewImage] = useState(null);
+
+  const onSubmit = async (data) => {
+    console.log(data.communityArtwork[0]);
```
Comment (Megha-Dev-19) [inline src/components/pages/landing/communityApplicationPanel/index.js]: do we need the log?

File: src/components/pages/landing/communityApplicationPanel/index.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+          )}
+        </div>
+        <div className="py-2">
+          <button
+            className="bg-blue-500 w-60 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline"
+            type="submit"
+            disabled={formLoading}
+          >
+            {formLoading
+              ? 'Applying , Please wait :)'
```
Comment (Megha-Dev-19) [inline src/components/pages/landing/communityApplicationPanel/index.js]: why do we have it `Please wait :`

File: src/pages/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import 'regenerator-runtime/runtime';
+import React, { useState } from 'react';
+
+import { Landing } from './unAuth/index';
+import { Home } from './auth/home';
+//hard code 3 near address to show additional data
```
Comment (Megha-Dev-19) [inline src/pages/index.js]: Can you explain what it means?

File: src/components/pages/landing/communityApplicationPanel/index.js [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+      toast.success('Applied for community successfully');
+    } finally {
+      setFormLoading(false);
+      setOpen(false);
+      reset();
+      fetchCommunities();
+    }
+
+    // await axios.post(`${apiLink}/`);
+    //
```
Comment (Megha-Dev-19) [inline src/components/pages/landing/communityApplicationPanel/index.js]: can we remove it?

File: src/components/common/footer.js [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+    { name: 'Jobs', href: '#' },
+    { name: 'Press', href: '#' },
+    { name: 'Partners', href: '#' },
+  ],
+  legal: [
+    { name: 'Claim', href: '#' },
+    { name: 'Privacy', href: '#' },
+    { name: 'Terms', href: '#' },
+  ],
+  social: [
```
Comment (Megha-Dev-19) [inline src/components/common/footer.js]: where do we show this social?
I am not able to check it on UI

**Review by Megha-Dev-19 — CHANGES_REQUESTED** [review]

**Comment by Megha-Dev-19** [comment]
image
This looks weird on smaller screens

- When I click on "Community Application for SBT", it loads to the page, but since I m at bottom I don't realise the above contents have changed, we should scroll to the top 

- There are no margins here (and loading text was a checkbox I guess)
image
image

File: src/components/pages/landing/communityApplicationPanel/index.js (hunk 2 above)
Comment (harjaapdhillon16) [reply src/components/pages/landing/communityApplicationPanel/index.js]: We don't really need it removing it !

File: src/components/pages/landing/communityApplicationPanel/index.js (hunk 4 above)
Comment (harjaapdhillon16) [reply src/components/pages/landing/communityApplicationPanel/index.js]: Yeah sure !

File: src/pages/index.js (hunk 3 above)
Comment (harjaapdhillon16) [reply src/pages/index.js]: Not relevant anymore removing it

File: src/components/pages/landing/communityApplicationPanel/index.js (hunk 1 above)
Comment (harjaapdhillon16) [reply src/components/pages/landing/communityApplicationPanel/index.js]: no not really removing it

File: src/components/common/footer.js (hunk 5 above)
Comment (harjaapdhillon16) [reply src/components/common/footer.js]: This code is not really being used right now , removing

**Comment by harjaapdhillon16** [comment]
@Megha-Dev-19 I have made all the fixes , if everything seems right , please approve and merge !

File: src/pages/communtiyapplication.js [hunk 6]
```diff
@@ -1,12 +1,18 @@
+import React from 'react';
 import { Footer } from '../components/common/footer';
 import { Header } from '../components/common/header';
 import { CommunitiesApplication } from '../components/pages/landing/communitiesApplication';
 
 export const CommunityApplicationPage = ({ isSignedIn }) => {
+  const ref = React.useRef();
+
+  React.useEffect(() => {
+    document.body.scrollTop = document.documentElement.scrollTop = 0;
```
Comment (Megha-Dev-19) [inline src/pages/communtiyapplication.js]: window.scrollTo(0, 0)
this should work
I don't think we need to assign ref for this

**Comment by Megha-Dev-19** [comment]
image
this loading looks weird 
may be we can have spinner with this, instead of this radio button looking element

**Comment by Megha-Dev-19** [comment]
Pre approving, left some comments

**Review by Megha-Dev-19 — APPROVED** [review]

**Comment by harjaapdhillon16** [comment]
> image this loading looks weird may be we can have spinner with this, instead of this radio button looking element

Change the color of the outline of the loader , looks better now

**Pull request merged**